src/lifetracker/UI/UIController.java

```diff
@@ -0,0 +1,186 @@
+package lifetracker.UI;
+
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.value.ObservableValue;
+import javafx.beans.property.ReadOnlyStringWrapper;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Label;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.control.TableColumn.CellDataFeatures;
+import javafx.util.Callback;
+import lifetracker.logic.ExecuteResult;
+import lifetracker.logic.Logic;
+
+public class UIController implements Initializable {
+
+    private static Logic l;
+
+    @FXML TextField textInput;
+    @FXML Label labelFeedback;
+    @FXML TableView<ItemUI> tableTask;
+    @FXML TableColumn<ItemUI, String> columnTaskID;
+    @FXML TableColumn<ItemUI, String> columnTaskName;
+    @FXML TableColumn<ItemUI, String> columnTaskTime;
+    @FXML TableView<ItemUI> tableEvent;
+    @FXML TableColumn<ItemUI, String> columnEventID;
+    @FXML TableColumn<ItemUI, String> columnEventName;
+    @FXML TableColumn<ItemUI, String> columnEventStartTime;
+    @FXML TableColumn<ItemUI, String> columnEventEndTime;
+    
+    private static ObservableList<ItemUI> taskList = FXCollections.observableArrayList();
+    private static ObservableList<ItemUI> eventList = FXCollections.observableArrayList();
+    
+    @FXML
+    public void getInput() {
+        String userInput;
+
+        if (!textInput.getText().toLowerCase().equals("exit")) {
```

Comment:
don't do a double parse here, logic handles the exit. Instead look out for the result.getType() == ExecuteResult.CommandType.EXIT